templates/user/dashboard/issues.tmpl

```diff
@@ -99,6 +99,7 @@
 								<a class="{{if eq .SortType "farduedate"}}active{{end}} item" href="{{$.Link}}?type={{$.ViewType}}&repos=[{{range $.RepoIDs}}{{.}}%2C{{end}}]&sort=farduedate&state={{$.State}}&q={{$.Keyword}}">{{.locale.Tr "repo.issues.filter_sort.farduedate"}}</a>
 							</div>
 						</div>
+						{{template "user/dashboard/issues_create" .}}
```

Comment:
Is there any need to be a template? Otherwise just have this code in here.

Reply:
Okay, I thought it's better organised.

Reply:
Hmm for a few lines that won't be re-used? Not really necessary to be a template.